### PR TITLE
feat(web-analytics): Add web analytics to onboarding

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem

Web analytics doesn't have an onboarding flow yet

## Changes

Add an onboarding flow! I haven't set up the product intro or the billing pages yet, as those changes live in the billing repo.

Not sure which it makes sense to merge first, but this one does work standalone, so potentially this one!

Screenshot 2024-10-02 at 13 44 05
Screenshot 2024-10-02 at 13 44 11
Screenshot 2024-10-02 at 13 44 15
Screenshot 2024-10-02 at 13 44 19

## Does this work well for both Cloud and self-hosted?

Yes

### CODE REVIEW

File: frontend/src/scenes/onboarding/sdks/web-analytics/FinalSteps.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <JSEventSnippet />
+        </>
+    )
+}
+
+export const WebAnalyticsMobileFinalSteps = (): JSX.Element => {
+    return (
+        <>
+            <h3>Track screen views</h3>
+            <p>Despite the name, the web analytics dashboard can be used to track screen views in mobile apps, too.</p>
```
Comment (raquelmsmith) [inline frontend/src/scenes/onboarding/sdks/web-analytics/FinalSteps.tsx:28]: Do they need to do anything with this info? Maybe open their app and view some screens to generate some events?

File: frontend/src/scenes/onboarding/Onboarding.tsx [hunk 2]
```diff
@@ -185,6 +186,77 @@ const ProductAnalyticsOnboarding = (): JSX.Element => {
...
+            visible: true,
+        },
+        {
+            title: 'Capture console logs',
+            description: `Automatically enable console log capture`,
+            teamProperty: 'capture_console_log_opt_in',
+            value: true,
+            type: 'toggle',
+            visible: false,
+        },
```
Comment (raquelmsmith) [inline frontend/src/scenes/onboarding/Onboarding.tsx:238]: I feel like this one isn't necessary for people just getting started with web analytics, so I'd exclude

**Review by raquelmsmith — APPROVED** [review]
Looks good! Just two comments

File: frontend/src/scenes/onboarding/sdks/web-analytics/FinalSteps.tsx (hunk 1 above)
Comment (robbie-c) [reply frontend/src/scenes/onboarding/sdks/web-analytics/FinalSteps.tsx]: I added some extra text here, similar to what we say for web

File: frontend/src/scenes/onboarding/Onboarding.tsx (hunk 2 above)
Comment (robbie-c) [reply frontend/src/scenes/onboarding/Onboarding.tsx]: Done

**Pull request merged**